engine/src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -16,13 +16,19 @@ import scala.util.Try
   * for now
   */
 object CromwellBackend {
-  private val log = LoggerFactory.getLogger(getClass.getName)
   private var _backends: Option[Map[String, Backend]] = None
   private var _defaultBackend: Option[Backend] = None
 
-  def initBackends(backendTypes: List[String], defaultBackend: String, actorSystem: ActorSystem) = {
-    _backends = Option((backendTypes map { backendType => (backendType, Backend.from(backendType, actorSystem)) }).toMap)
-    _defaultBackend = Option(Backend.from(defaultBackend, actorSystem))
+  def backendFromConfig(entry: BackendConfigurationEntry, actorSystem: ActorSystem): Backend = {
+    val constructor = Class.forName(entry.className).getConstructor(classOf[BackendConfigurationEntry], classOf[ActorSystem])
+    constructor.newInstance(entry, actorSystem).asInstanceOf[Backend]
```

Comment:
Yes I think that's right, there has to be some contract for how to instantiate a backend.